.dict_custom.txt

```diff
@@ -119,3 +119,4 @@ indexable
 traceback
 STC
 gFTL
+Optimize
```

Comment:
optimize should be in the dictionary. Probably it is missing because we are using UK english

Reply:
Shall I remove it and use `optimise` instead?

Reply:
Yes please

Reply:
done